.github/workflows/tests.yml

```diff
@@ -219,6 +219,10 @@ jobs:
           sudo apt-get update
 
           sudo snap remove lxd --purge
+          # Purge disabled (superseded) snaps
```

Comment:
Could you expand the comment on what this does please m its not clear what this is removing?

Reply:
Will do but briefly, it's removing all the older snap revs, those ones:
```
$ snap list --all | grep -wF disabled
go                                     1.22-1ae729e      10478  latest/stable    canonical**     disabled,classic
lxd                                    git-f3383ec       26941  latest/edge      canonical**     disabled
```

Reply:
By older revs, I mean those superseded by newer revs (like fresher lxd/go in my case).